core/src/apps/bitcoin/sign_tx/approvers.py

```diff
@@ -42,6 +42,10 @@ def __init__(self, tx: SignTx, coin: CoinInfo) -> None:
         self.orig_total_out = 0  # sum of original output amounts
         self.orig_change_out = 0  # sum of original change output amounts
 
+        self.amount_unit = (
```

Comment:
let's rely on the default from protobuf

Reply:
Fix in 3a1b98050463c515587349cf6abbb4090d6ee8ab